sql/core/src/test/scala/org/apache/spark/sql/execution/adaptive/AdaptiveQueryExecSuite.scala

```diff
@@ -899,4 +900,16 @@ class AdaptiveQueryExecSuite
         checkPlanDescription(mode, expected)
     }
   }
+
+  test("SPARK-30953: InsertAdaptiveSparkPlan should apply AQE on child plan of write commands") {
+    withSQLConf(SQLConf.ADAPTIVE_EXECUTION_ENABLED.key -> "true") {
+      spark.range(10).repartition(1).selectExpr("id as a", "id * 2 as b")
+        .createOrReplaceTempView("t0")
+      val df = spark.sql("CREATE TABLE t1 AS SELECT a FROM t0")
```

Comment:
we can call `sql(....)` directly.

Reply:
for simplicity, how about `sql("CREATE TABLE t1 AS SELECT 1 col")`?

Reply:
ok.